performance-test/scripts/Resident_Test_script.jmx

```diff
@@ -0,0 +1,8333 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="Test Plan" enabled="true">
+      <stringProp name="TestPlan.comments"></stringProp>
+      <boolProp name="TestPlan.functional_mode">false</boolProp>
+      <boolProp name="TestPlan.tearDown_on_shutdown">true</boolProp>
+      <boolProp name="TestPlan.serialize_threadgroups">false</boolProp>
+      <elementProp name="TestPlan.user_defined_variables" elementType="Arguments" guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments"/>
+      </elementProp>
+      <stringProp name="TestPlan.user_define_classpath"></stringProp>
+    </TestPlan>
```

Comment:
If we are using the same access token for Logout Endpoint, how will the same user be used again ?

Reply:
I am able to use the same access token for other activities even after logout is completed in the script.

Reply:
Can we get this flow verified from the module lead/developer ?

Reply:
While verifying with the dev team, we found this as a bug. I have created a ticket for the same.

Reply:
@vishwa-vyom Can I remove logout API from the script till the bug is resolved?

Reply:
Added the setup for creating access_tokens for logout scenario, and these will only be used for logout API.

Reply:
if this work is specific to a thread group, then it should be a preparation thread group and not setup

Reply:
Also readme file to be updated with all the details for this scenario

Reply:
This is now part of preparation thread group, also readme file is updated with the details of this scenario.

Reply:
Reviewed.

Reply:
Normally prepartion and execution thead groups are a pair, so the same thread group name to be maintained, only change should be (preparation) instead of (execution)
Lets mention this in the develop guide also

Reply:
Changed the naming convention as per execution thread group. Also updated the developer guide.

Reply:
Reviewed. S10 Profile Icon & Logout (Preparation) has been updated.